src/server/console/logind_console_services.cpp

```diff
@@ -43,21 +44,23 @@ class GErrorPtr
         : raw_error{nullptr}
     {
     }
-    GErrorPtr(GErrorPtr const&) = delete;
-    GErrorPtr& operator=(GErrorPtr const&) = delete;
+
+    GErrorPtr(GErrorPtr const &) = delete;
+
+    GErrorPtr &operator=(GErrorPtr const &) = delete;
 
     ~GErrorPtr()
     {
         if (raw_error)
             g_error_free(raw_error);
     }
 
-    GError** operator&()
+    GError **operator&()
     {
         return &raw_error;
     }
 
-    GError* operator->()
+    GError *operator->()
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style